packages/cubejs-docker/README.md

```diff
@@ -1,25 +1,100 @@
 <p align="center"><a href="https://cube.dev"><img src="https://i.imgur.com/zYHXm4o.png" alt="Cube.js" width="300px"></a></p>
 
-[Website](https://cube.dev) • [Docs](https://cube.dev/docs) • [Blog](https://cube.dev/blog) • [Slack](https://slack.cube.dev) • [Twitter](https://twitter.com/thecubejs)
+[Website](https://cube.dev) • [Docs](https://cube.dev/docs) •
+[Blog](https://cube.dev/blog) • [Slack](https://slack.cube.dev) •
+[Twitter](https://twitter.com/thecubejs)
 
 # Supported tags and respective `Dockerfile` links
 
-- [latest](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest.Dockerfile) - Latest stable release based on Debian (recommended)
-- [alpine](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest-alpine.Dockerfile) - Latest stable release on Alpine
-- [dev](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/dev.Dockerfile) - Latest development release from master branch.
+- [latest](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest.Dockerfile) -
+  Latest stable release, based on Debian (recommended)
+- [alpine](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest-alpine.Dockerfile) -
+  Latest stable release on Alpine
+- [dev](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/dev.Dockerfile) -
+  Latest development release from master branch.
+
+# Quick reference
+
+Where to get help? Check out our Slack, Stack Overflow or GitHub Discussions
```

Comment:
Should there be links for our Slack, GH Discussions/Issues?

Reply:
Whoops 😅 great catch @rpaik 👍